deploy/bring-your-own-ai-model.md

```diff
@@ -0,0 +1,69 @@
+## Using AI-Models
```

Comment:
Can we keep the deploy folder less simplified and with less files? I feel there are too many files under this folder. We should instead move this markdown file to docs. 

@mandarinamdar  Thoughts?

Reply:
I would agree on this. May be another change should address this.
We can move the script files to scripts directory and create a docs directory in here to keep context related documents with README pointing to those files.

Reply:
I am fine it getting addressed in a separate PR

Reply:
created the issue https://github.com/Azure/Azure-Orbital-Analytics-Samples/issues/91

Reply:
Lets create a issue for this and handle it in separate PR

Reply:
Issue created